boot/bootdata/hivecls.inf

```diff
@@ -849,6 +849,18 @@ MIDFILE="Séquence MIDI"
 CDAFILE="Piste de CD Audio"
 AVIFILE="Clip Vidéo"
 
+CSSFILE="Document de feuille de style en cascade"
```

Comment:
Wouldn't be "Fichier CSS" be shorter and easier to read ?

Reply:
OK